open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/geom/style/labelcontroller/LabelHoverController.java

```diff
@@ -99,6 +102,22 @@ public void notify(DataElementHighlightChangeEvent event)
         }
     }
 
+    /**
+     * Determines whether or not the geometry we're trying to highlight is in an
+     * activated state.
+     *
+     * @param event the highlight event
+     * @return whether the data group containing the event element is active
+     */
+    private boolean isEventElementActive(DataElementHighlightChangeEvent event)
+    {
+        MantleToolbox toolbox = MantleToolboxUtils.getMantleToolbox(getToolbox());
+        String dtiKey = event.getDataTypeKey();
+        DataGroupActivationProperty activation = toolbox.getDataGroupController().getDataGroupInfo(dtiKey).activationProperty();
+
+        return activation.getValue() == ActivationState.ACTIVE;

```

Comment:
I don't know the answer to this but think about if we should also consider groups in ACTIVATING as active.  If so, there's a DataGroupActivationProperty.isActiveOrActivating() you can use.

Reply:
I guess we wouldn't have features to hover over if we're activating, to just checking ACTIVE is probably fine.